src/QMCWaveFunctions/LCAO/SoaLocalizedBasisSet.h

```diff
@@ -213,6 +213,7 @@ class SoaLocalizedBasisSet : public SoaBasisSetBase<ORBT>
   struct SoaLocalizedBSetMultiWalkerMem;
   /// multi walker resource handle
   ResourceHandle<SoaLocalizedBSetMultiWalkerMem> mw_mem_handle_;
+  NewTimer & NumCenter_timer_ ;
```

Comment:
I don't feel the need of adding this timer. Those inside `LOBasisSet[IonID[c]]->mw_evaluateVGL` is probably enough.

Reply:
I will leave as I need it for the next one and let's not be pedantic on one timer I really need for the next PR

Reply:
If it is needed for the next one, it is better to be included in the next one. We keep PRs focusing on self contained topics.